### ISSUE TITLE
Websocket bypasses proxy support

### ISSUE THREAD

**Issue by chrmarti** [microsoft/vscode-azure-account#111]
@chrmarti, the Cloud Shell seems to be quite buggy at the moment.
It worked only once, that I could see the whole Azure Cloud Shell web page / command line interface. But I couldn't get it work again thereafter.
It keeps giving me the error now (after the second try and onward)
`Requesting a Cloud Shell...
Connecting terminal...
Socket error: {"errno":"EPROTO","code":"EPROTO","syscall":"write"}
Socket closed`

 
> None of the other extensions are using the new proxy support yet, that's why you see these other commands fail as before.

When can we expect the other extensions to use the new proxy support? Thanks!

_Originally posted by @revolz in https://github.com/Microsoft/vscode-azure-account/issues/27#issuecomment-440211135_

**Comment by wwlorey** [comment]
Closing as duplicate of https://github.com/microsoft/vscode-azure-account/issues/179
